romancal/pipeline/highlevel_pipeline.py

```diff
@@ -55,13 +58,18 @@ def process(self, input):
             return
 
         if file_type == "asn":
+            input = ModelContainer(input)
+            self.flux.suffix = "flux"
+            result = self.flux(input)
             self.skymatch.suffix = "skymatch"
             result = self.skymatch(input)
```

Comment:
```suggestion
            result = self.skymatch(result)
```

Reply:
You have good eyes!

Reply:
Thanks, I did not update that step after adding the flux step in, done.